src/contracts.ts

```diff
@@ -24,6 +24,11 @@ export class Contracts {
         contractAddresses = contractAddresses ?? <RelayingServicesAddresses>{};
         const contracts: RelayingServicesAddresses =
             getContractAddresses(chainId);
+        if (!contracts) {
```

Comment:
I think that raising an error here won't allow the user to initialize the `Contracts` object.
We could move the error (maybe creating a custom error) in the `getContractAddresses` and catch the error here (maybe adding some warnings) to be able to complete the `Contracts` initialization.